de_web_plugin.cpp

```diff
@@ -7586,6 +7598,12 @@ void DeRestPluginPrivate::updateSensorNode(const deCONZ::NodeEvent &event)
 
                             if (ia->id() == 0x0021) // battery percentage remaining
                             {
+                                // filter endpoint
+                                if (event.endpoint() != i->fingerPrint().endpoint)
+                                {
+                                    continue;
+                                }
+
```

Comment:
This is so that only the RConfigItem belonging to the endpoint in question is written. Otherwise, all the endpoints end up reporting the same value - the one for the last-read endpoint.

I'm not too familiar with the project, if there's a better way to accomplish this, please let me know!